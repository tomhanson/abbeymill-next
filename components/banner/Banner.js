// @flow strict

import * as React from 'react';
import { BannerStyled, BannerHeadline } from './Banner.styles';

type Props = {
  children: React.Node,
  img: string,
};

const Banner = ({ children, img }: Props) => (
  <BannerStyled img={img}>
    <BannerHeadline>{children}</BannerHeadline>
  </BannerStyled>
);

export default Banner;
