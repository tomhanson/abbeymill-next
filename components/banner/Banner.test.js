import { render } from 'react-testing-library';
import Banner from './Banner';

describe('Banner at the top of the page', () => {
  test('the component loads correctly', () => {
    const component = render(<Banner />);
    expect(component).toMatchSnapshot();
  });
});
