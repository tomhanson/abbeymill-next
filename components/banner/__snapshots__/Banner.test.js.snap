// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner at the top of the page the component loads correctly 1`] = `
Object {
  "baseElement": <html>
    <head>
      <style
        data-styled-components=""
      >
        
        
/* sc-component-id: sc-bdVaJa */
.sc-bdVaJa {} .ixlWdJ{background-size:cover;background-repeat:no-repeat;background-image:url('');position:relative;background-position:50%;height:95vh;max-height:670px;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;} .ixlWdJ:before{content:"";position:absolute;top:0;right:0;bottom:0;left:0;background-color:rgba(74,74,74,.4);}
        
/* sc-component-id: sc-bwzfXH */
.sc-bwzfXH {} .gJIwTT{text-align:center;}
      </style>
    </head>
    <body>
      <div>
        <section
          class="sc-bdVaJa ixlWdJ"
        >
          <div
            class="sc-bwzfXH gJIwTT"
          />
        </section>
      </div>
    </body>
  </html>,
  "container": <div>
    <section
      class="sc-bdVaJa ixlWdJ"
    >
      <div
        class="sc-bwzfXH gJIwTT"
      />
    </section>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
